<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>898</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RousettusTool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size:10pt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="horizontalFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Project name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_prj_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_prj_path">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListView" name="logger_view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="data_processing_menu">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Data Processing</string>
    </property>
    <widget class="QMenu" name="menuMagnetic_Data">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Magnetic Data</string>
     </property>
     <addaction name="actionVariation_calculate"/>
    </widget>
    <addaction name="menuMagnetic_Data"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="OpenPrjButton"/>
    <addaction name="separator"/>
    <addaction name="exitButton"/>
   </widget>
   <widget class="QMenu" name="settings_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="DBsettingsButton"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuFligft_Planning">
    <property name="title">
     <string>Fligft Planning</string>
    </property>
    <addaction name="actionMake_Profiles"/>
    <addaction name="actionPlan_routes"/>
    <addaction name="actionplan_test"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Rousettus"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="data_processing_menu"/>
   <addaction name="menuFligft_Planning"/>
   <addaction name="settings_menu"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="exitButton">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="DBsettingsButton">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Database settings</string>
   </property>
  </action>
  <action name="action_CSV">
   <property name="text">
    <string>из файла CSV</string>
   </property>
  </action>
  <action name="OpenPrjButton">
   <property name="text">
    <string>Открыть проект</string>
   </property>
  </action>
  <action name="actionVariation_calculate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Variation calculate</string>
   </property>
  </action>
  <action name="actionMake_Profiles">
   <property name="text">
    <string>Make Profiles</string>
   </property>
  </action>
  <action name="actionPlan_routes">
   <property name="text">
    <string>Plan Routes</string>
   </property>
  </action>
  <action name="actionAbout_Rousettus">
   <property name="text">
    <string>About Rousettus</string>
   </property>
  </action>
  <action name="actionTest_plan_flight">
   <property name="text">
    <string>Test_plan_flight</string>
   </property>
  </action>
  <action name="actionplan_test">
   <property name="text">
    <string>Plan Flights</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
